docs/guides/manage_serverless_resources.ipynb

```diff
@@ -0,0 +1,334 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# source_files/transpile_parallel.py\n",
+    "\n",
+    "from qiskit.transpiler.preset_passmanagers import generate_preset_pass_manager\n",
+    "from qiskit_serverless_tools.selectors import IBMLeastBusyQPUSelector\n",
+    "\n",
+    "backend = IBMLeastBusyQPUSelector(service).get_backend(min_num_qubits=127)\n",
+    "pm = generate_preset_pass_manager(optimization_level=3, backend=backend)\n",
+    "target_circuit = pm.run(abstract_circuit)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Managing data across your program\n",
+    "\n",
+    "Qiskit Serverless allows you to manage files in `/data` directory across all your programs. This includesd several limitations:\n",
+    "\n",
+    "- Only `tar` and `h5` files are supported today\n",
+    "- This is only a flat `/data` storage, and cannot have `/data/folder/` subdirectories\n",
+    "\n",
+    "First, we'll explore how to upload files:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit_serverless import IBMServerlessClient\n",
+    "\n",
+    "# Authenticate to the remote cluster and submit the pattern for remote execution\n",
+    "serverless = IBMServerlessClient(\n",
+    "    token=API_TOKEN\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'{\"message\":\"/usr/src/app/media/5f37582aa306c50013fac285/transpile_demo.tar\"}'"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import tarfile\n",
+    "\n",
+    "# Create a tar\n",
+    "filename = \"transpile_demo.tar\"\n",
+    "file = tarfile.open(filename,\"w\")\n",
+    "file.add(\"source_files/transpile_remote.py\")\n",
+    "file.close()\n",
+    "\n",
+    "# Upload the tar to Serverless data directory\n",
+    "serverless.file_upload(filename)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next, we can list all the files in our `data` directory. This data is accessible to all programs."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "['transpile_demo.tar']"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "serverless.files()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This can be used from a program by using `file_download()` to download the file to the program environment, and uncompressing the `tar`"
```

Comment:
```suggestion
    "This can be used from a program by using `file_download()` to download the file to the program environment, and uncompressing the `tar`."
```